zng/alias.go

```diff
@@ -45,3 +45,12 @@ func (t *TypeAlias) StringOf(zv zcode.Bytes, out OutFmt, b bool) string {
 func (t *TypeAlias) Marshal(zv zcode.Bytes) (interface{}, error) {
 	return t.Type.Marshal(zv)
 }
+
+func AliasedType(typ Type) Type {
```

Comment:
Would you consider naming this `Unalias`? I find that would be a little clearer, and would stutter less in its usage:
`zng.Unalias(val.Type).(type)` 
`zng.Unalias(v.Type) != zng.TypeFoo`

Reply:
This may be moot with the discussion of the `val.Type.ID()` usage below, but fyi, using `Unalias` came to mind partly by how go errors are unwrapped, both in the popular `pkg/errors` package from Dave Cheney, and now via the go standard library:
https://godoc.org/github.com/pkg/errors#Unwrap
https://golang.org/pkg/errors/#Unwrap